src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/DataFlowPass.vb

```diff
@@ -615,13 +615,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             ' If the state of the slot is 'assigned' we need to clear it 
             '    and possibly propagate it to all the parents
+            If Me._tryState IsNot Nothing Then
```

Comment:
Actually that was not entirely correct. unassignments are also used for region analysis and that can have partially assigned/unassigned structs. So for completeness we do need to propagate unassignments of parent variables to the tryState too. I cant figure right away what kind of testcase would hit this.